src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -465,6 +505,53 @@ private void ParseRuleFromDescription()
             _ruleDictionary.Add(rule.Id, rule);
         }
 
+        private void ParseNodes()
+        {
+            _reader.Read();
+            while (!AtEndOf(_strings.UnifiedNodePool))
+            {
+                if (AtStartOfNonEmpty(_strings.Node))
```

Comment:
>if [](start = 16, length = 2)

I believe this 8 line vertically spaced code sequence reduces to a single line, if you don't mind it stylistically.

AtStartOfNonEmpty(_strings.Node) ? ParseNode() : _reader.Read()

Reply:
That syntax only works for assignment.

---
In reply to: [207321311](https://github.com/Microsoft/sarif-sdk/pull/981#discussion_r207321311) [](ancestors = 207321311)